src/persist-client/src/internal/state_serde.json

```diff
@@ -3,488 +3,498 @@
   "shard_id": "s0496ebd7-5c07-f7cd-1499-e74ed5c9bb03",
   "seqno": 1773488203606603407,
   "walltime_ms": 12135341112053470409,
-  "hostname": "𞸴ᜲ&ã/𝋊{'`𐖩%<ֈⶻ2 ﹪¥9>𝔚𞸁𖨓=e&᳆ \\",
```

Comment:
@bkirwi Is this diff fine? It's the main change I had to make other than surrounding infrastructure.